src/Spatial.Tests/Euclidean/Line2DTests.cs

```diff
@@ -56,6 +56,38 @@ public void LineDirection(string p1s, string p2s, string exs)
             AssertGeometry.AreEqual(ex, line.Direction);
         }
 
+        public static object[] DistanceTestCases =
+        {
+            new object[] { "-1,+1", "+1,+1", "0,0", 1.0 },
+            new object[] { "-1,+1", "+1,+1", "0,3", 2.0 },
+            new object[] { "-1,+1", "+1,+1", "0,1", 0.0 },
+            new object[] { "-1,0", "0,+1", "0,0", 0.70710678 },
+            new object[] { "-1,0", "0,+1", "-0.5,0.5", 0.0 },
+            new object[] { "+1,-1", "+1,+1", "0,0", 1.0 },
+            new object[] { "+1,-1", "+1,+1", "1,0", 0.0 },
+            new object[] { "+1,-1", "+1,+1", "3,0", 2.0 },
+        };
+
+        [TestCaseSource(nameof(DistanceTestCases))]
```

Comment:
I think the previous structure with `TestCase` was much better. I think we can test the swapped scenarios within the same test function, there is no need to define new tests.

Reply:
Thanks for the comment.
Could you tell me why using `TestCode` is better than using `TestCodeSource` for this test?

Reply:
I find the whole `new object[]` a bit overkill, that's it. I guess it's more a matter of personal taste.

Reply:
Thanks for the comment.

> it's more a matter of personal taste.

Could you show the reason?

Reply:
The reason for what exactly? I prefer compile-time checks and try to avoid `object` as much as possible

Reply:
Thanks for telling the reasons.

Done in the commit https://github.com/mathnet/mathnet-spatial/pull/238/commits/cd80cf195a07cb3a79f0ccf35dad3f26f14cc89e.
I changed the test-code as you required. I'm not sure that the revised test-code helps the reader understand that parameters are kept intact except its order at a glance.

Reply:
Thank you. This is not what I meant, we can augment the existing tests with a reversed line
```
var line = new Line2D(Point2D.Parse(p1s), Point2D.Parse(p2s));
// existing test

var reversedLine = new Line2D(Point2D.Parse(p2s), Point2D.Parse(p1s));
// distance test with reversed line
```

This way you keep the `TestCase` cases at a minimum

Reply:
Thanks for showing the example closely.

It is not recommended to test 2 things in a single unit-test.
Or do you want to say that these 2 tests are equivalent because of the symmetry?

Reply:
> It is not recommended to test 2 things in a single unit-test. 

Fully agree

> Or do you want to say that these 2 tests are equivalent because of the symmetry?

Basically yes